entities/params/templates.go

```diff
@@ -4,9 +4,10 @@ import "strings"
 
 // PostTemplate represents request body for POST /api/templates
 type PostTemplate struct {
-	Name    string `form:"name" validate:"required,max=191"`
-	Content string `form:"content" validate:"required,html"`
-	Subject string `form:"subject" validate:"required,max=191"`
+	Name     string `form:"name" validate:"required,max=191"`
+	HTMLPart string `form:"html_part" validate:"required,html"`
+	TextPart string `form:"text_part" validate:"required"`
```

Comment:
didnt we said that `TextPart` should be optional?

Reply:
No